internal/pkg/skuba/deployments/ssh/ssh.go

```diff
@@ -237,6 +243,14 @@ func (t *Target) initClient() error {
 			klog.Errorf("ssh authentication error: please make sure you have added to "+
 				"your ssh-agent a ssh key that is authorized in %q.", t.target.Target)
 			return errSSHAuthErr
+		} else if strings.Contains(err.Error(), "connection refused") {
+			klog.Errorf("ssh error: please make sure you have the correct IP/FQDN to "+
```

Comment:
Add port information on klog output would be better if it's denied by firewall.

Reply:
Maybe also mention "and ssh is running on port $port" or similar in that message?

Reply:
That is out of scope for this card. It can be tackled separate issue cards something like  ssh detailed logs. I added this log messages because I used previously deployed IP address on Skuba cmd. Therefore I had the error message with "connection refused" and I spent some time without knowing the reason. So, I added this detailed log error message.

To tackle the detailed ssh connection error, we need find error message from GO ssh. I am not sure what error messages Go ssh will give for each of different connection errors. So this can be tackled different card or everyone can add when it is needed.

Reply:
I would like to saw port number in `klog.Errorf` output message, that's it.

Reply:
I added it.